packages/web-pkg/src/composables/actions/files/useFileActionsToggleHideShare.ts

```diff
@@ -41,19 +33,17 @@ export const useFileActionsToggleHideShare = () => {
       triggerPromises.push(
         triggerQueue.add(async () => {
           try {
-            const share = await triggerShareAction({
+            await triggerShareAction({
               resource,
-              status: resource.status,
+              status: resource.syncEnabled ? 0 : 2,
```

Comment:
This is probably for the time being until we're moving towards the graph client completely?

Reply:
This will be removed as one of the next steps, so again I decided to not invest much time into making it more readable.